.github/workflows/conformance-ingress.yaml

```diff
@@ -91,12 +91,15 @@ jobs:
           fi
           echo sha=${SHA} >> $GITHUB_OUTPUT
 
-      - name: Checkout
+      - name: Checkout context ref
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
         with:
-          ref: ${{ steps.vars.outputs.sha }}
+          ref: ${{ inputs.context-ref || github.sha }}
           persist-credentials: false
 
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
```

Comment:
You can drop this commit altogether, as https://github.com/cilium/cilium/commit/fd2240b8e3edd697277de1da22d6822270d403ee was never backported to v1.12.